admin/index.php

```diff
@@ -227,7 +227,7 @@
 		$display_on_page = ( $offset + 1 );
 	}
 
-	// Determing Total Amount Of Pages
+	// Determining Total Amount Of Pages
```

Comment:
I see where you came from with the existing change, but in context (and based on the other comments above) this is the intended wording:

```suggestion
	// Determine Total Amount Of Pages
```

Reply:
Agreed.  Updated to use your suggestion.